apps/archive/commands.py

```diff
@@ -48,6 +52,14 @@ def run(self):
 
         remove_locks()
 
+    def _remove_expired_publish_queue_items(self):
+        expire_interval = app.config.get('PUBLISH_QUEUE_EXPIRY_TIME', 0)
+        if expire_interval:
+            expire_time = utcnow() - timedelta(seconds=expire_interval)
```

Comment:
imo would be good to be consistent with the other options so the config should be in minutes

Reply:
like `PUBLISH_QUEUE_EXPIRY_MINUTES`

Reply:
Then I would have to set the sleep time for the behave test to 61 seconds, otherwise it wouldn't work.

Reply:
I think you can set an _id manually when you post to publish queue

Reply:
But this has nothing to do with _id

Reply:
I agreed with @petrjasek regarding the name

Probably you can use `PUBLISH_QUEUE_EXPIRY_MINUTES = -1` for tests like https://github.com/superdesk/superdesk-core/blob/master/tests/audit_purge_test.py#L16

Reply:
using this [superdesk.tests.steps.given_config_update](https://github.com/superdesk/superdesk-core/blob/master/superdesk/tests/steps.py#L376)

Reply:
if the _id is older it will pick it or am I missing something?